tests/gold_tests/tls/tls_client_versions.test.py

```diff
@@ -71,28 +71,28 @@
 tr = Test.AddTestRun("foo.com TLSv1_2")
 tr.Processes.Default.StartBefore(server)
 tr.Processes.Default.StartBefore(Test.Processes.ts)
-tr.Processes.Default.Command = "curl -v --tls-max 1.2 --tlsv1.2 --resolve 'foo.com:{0}:127.0.0.1' -k  https://foo.com:{0}".format(
+tr.Processes.Default.Command = "curl -v --ciphers DEFAULT@SECLEVEL=0 --tls-max 1.2 --tlsv1.2 --resolve 'foo.com:{0}:127.0.0.1' -k  https://foo.com:{0}".format(
```

Comment:
Please add a comment to explain why you need to set the ciphers flag, and to refer the reader to the docs that explain the syntax

https://www.openssl.org/docs/manmaster/man1/openssl-ciphers.html

Reply:
Sounds good. Comment added. I chose to link an explanation of why security level is being set rather than the syntax, because I figure the former is more relevant to the reader here and the syntax would be easier to find.